app/src/main/java/com/riyas/myapplication/login_validation.kt

```diff
@@ -0,0 +1,36 @@
+class LoginValidator {
+
+    fun validateEmail(email: String): String {
+        if (email.isBlank()) {
+            return "Email cannot be empty"
+        }
+        if (!android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()) {
+            return "Invalid email format"
+        }
+        return "Valid"
+    }
+
+    fun validatePassword(password: String): String {
```

Comment:
similar to email validation , the name has to be IsvalidPassword() . Can we kdoc this function?